ChangeLog

```diff
@@ -1,3 +1,65 @@
+* Thu Oct 15 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
+- Changesets: 497
+- Diff: 285 files changed, 22704 insertions(+), 14458 deletions(-)
+
+- Features added since Pacemaker-2.0.4
+  + configuration: Support integer as rule type attribute.
+  + daemons: Recognize new OCF agent status codes 190 (degraded) and 191
+             (degraded master) to be treated as success but displayed as
+             errors.
+  + sbd-integration: support SBD_SYNC_RESOURCE_STARTUP environment
+                     variable to better synchronize Pacemaker start-up
+                     and shutdown with SBD
+  + scheduler: Add type="integer" to rule elements, allowing for
+               specifying 64-bit integers.
```

Comment:
I just noticed we have the integer thing in there twice

Also, add something like "and specifying double-precision floating-point numbers when type="number"" (number was formerly handled as 32-bit integer)